Shaders/qUINT_common.fxh

```diff
@@ -77,7 +100,11 @@ namespace qUINT
 #if RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN
 		uv.y = 1.0 - uv.y;
 #endif
-		float depth = tex2Dlod(sDepthBufferTex, float4(uv, 0, 0)).x;
+		uv.x /= RESHADE_DEPTH_INPUT_X_SCALE;
+		uv.y /= RESHADE_DEPTH_INPUT_Y_SCALE;
+		uv.x -= RESHADE_DEPTH_INPUT_X_OFFSET / 2.000000001;
+		uv.y += RESHADE_DEPTH_INPUT_Y_OFFSET / 2.000000001;
+		float depth = tex2Dlod(DepthBuffer, float4(uv, 0, 0)).x * RESHADE_DEPTH_MULTIPLIER;
```

Comment:
how did i miss this one, ffs

Reply:
I updated the files locally already, so I'm closing this PR as the necessary changes are now already present :)